crates/sui-core/src/authority.rs

```diff
@@ -189,7 +189,8 @@ pub struct AuthorityMetrics {
     batch_size: Histogram,
 
     handle_transaction_latency: Histogram,
-    execute_certificate_latency: Histogram,
+    execute_certificate_consensus_latency: Histogram,
```

Comment:
Just use a label to differentiate between the two?

Reply:
I can do that if you prefer that, just curious any particular reason it is better?

Reply:
Just in case we need to add more labels for the metric. Also there is currently `handle_transaction_latency` so it seems nice to maintain `execute_certificate_latency`.

Reply:
not sure what you mean by `handle_transaction_latency` but I will update to use labels

Reply:
I wanted to use two metrics because today we have `handle_certificate_non_consensus_latency` 
and `handle_certificate_consensus_latency` which are two metrics. But it's not a big deal and I don't have preferences